model-server/src/test/kotlin/org/modelix/model/server/ModelClientV2Test.kt

```diff
@@ -46,6 +48,8 @@ class ModelClientV2Test {
                 json()
             }
             install(WebSockets)
+            install(Resources)
+            install(IgnoreTrailingSlash)
```

Comment:
This is pretty much repeated in many tests. Maybe a common abstraction would be good then.

Reply:
True, but I consider that part of a larger effort after this PR to restructure and extend tests. I will leave it as is for now.